pkg/azure/aks/aks.go

```diff
@@ -63,19 +63,19 @@ var (
 	InstanceCPUHourlyCostDesc = prometheus.NewDesc(
 		prometheus.BuildFQName(cloudcost_exporter.MetricPrefix, subsystem, "instance_cpu_usd_per_core_hour"),
 		"The cpu cost a compute instance in USD/(core*h)",
-		[]string{"instance", "region", "machine_type", "cluster_name", "price_tier", "operating_system"},
```

Comment:
I had a chat (live in person, I know the world is crazy) about the `price_tier` topic with @bastischubert. Some of our customers are very eager to get metrics about this (which OpenCost isn't doing). Maybe you can reach out to and we can find out if this attribute is already enough, or if they need something else.